src/Tasks/Microsoft.Build.Tasks.csproj

```diff
@@ -289,7 +289,6 @@
       <ExcludeFromStyleCop>true</ExcludeFromStyleCop>
     </Compile>
     <Compile Include="AssemblyDependency\Resolver.cs">
-      <SubType>Code</SubType>
```

Comment:
Where did we land on leaving/reverting these?

Reply:
VS removed it, and I don't particular want to fight VS. :-)

However I can just drop that file from the PR if you would prefer.